src/payment/strategies/paypal-commerce/paypal-commerce-payment-processor.ts

```diff
@@ -143,6 +133,19 @@ export default class PaypalCommercePaymentProcessor {
         return this._hostedFields.submit(options);
     }
 
+    validateHostedForm(): { isValid: boolean; fields: PaypalCommerceHostedFieldsState['fields'] } {
```

Comment:
`isValid` parameter is also an unused parameter.

Reply:
Hm, I use it [here](https://github.com/bigcommerce/checkout-sdk-js/pull/1003/files#diff-50215dc105323ccd925b612e6d34f5cea9d7908cf618b1fcd386faf238fc701aR68)

Reply:
Ah sorry I misread, it's a return value rather than a parameter. 👍